src/test/java/firstSimpleJavaTasks/ArrayEditingTests.java

```diff
@@ -0,0 +1,39 @@
+package firstSimpleJavaTasks;
+
+
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Parameterized.class)
```

Comment:
Please use data provider from this example https://github.com/TNG/junit-dataprovider/wiki/Getting-started

Reply:
I recommend writing a separate class using data-providers and compare the readability, limitations, and usability of both approaches.

Reply:
Separate class for unit tests that is using dataproviders was added